### ISSUE TITLE
IE11: Cannot interact with checkbox input with role="switch" if width/height 10px or less

### ISSUE THREAD

**Issue by xcindyzunic** [FreedomScientific/VFO-standards-support#390]
### Summary

> When JAWS 2020 is running in IE11, a checkbox with role="switch" cannot be checked/unchecked (via spacebar or enter key) if that checkbox has a width/height CSS property of 10px or less.
>When JAWS is not running, the same checkbox works as expected.
>No issues in Firefox or Chrome.
> 
> Example:
> https://www.w3schools.com/code/tryit.asp?filename=GF8JA8HNUJPZ

### Expected result

> Width/height should have no impact on checkbox (role="switch") function when JAWS is running.

### Actual result

> In IE11, you cannot check/uncheck the checkbox with role="switch" if that checkbox is smaller than 10px by 10px

### Example

> https://www.w3schools.com/code/tryit.asp?filename=GF8JA8HNUJPZ

#### JAWS version and build number
2020.2001.70

 
#### Operating System and version
Windows 10 Pro
Version 1903
OS build 18362.836

#### Browser and version:
Internet Explorer 11
Version: 11.836.18362.0
Update Versions: 11.0.190 (KB4556798)

**Comment by JAWS-test** [comment]
I can confirm the bug. The checkboxes can only be activated via their label. If the checkbox is activated, there is a loss of focus to the top of the page.

**Comment by stevefaulkner** [comment]
Closing this issue as wontfix as its IE only
